universal-login-react/test/config/setupGlobalFetch.ts

```diff
@@ -0,0 +1 @@
+(global as any).fetch = require('node-fetch');
```

Comment:
It looks pretty similar to [this](https://github.com/UniversalLogin/UniversalLoginSDK/blob/master/universal-login-wallet/test/integration/helpers/setupCryptoCompare.ts). Maybe we should merge it?

Reply:
extract setupGlobalFetch and deduplicate